lib/src/voip/backend/mesh_backend.dart

```diff
@@ -20,7 +20,8 @@ class MeshBackend extends CallBackend {
   /// participant:volume
   final Map<CallParticipant, double> _audioLevelsMap = {};
 
-  StreamSubscription<CallSession>? _callSubscription;
+  StreamSubscription<CallSession>? _callSetupSubscription;
```

Comment:
can you please add comments to what each of these streams is for?

Reply:
done

Reply:
_callSetupSubscription and _callStartSubscription still seem to be missing a  comment

Reply:
done